tftb/processing/linear.py

```diff
@@ -163,11 +165,28 @@ def gabor(signal, n_coeff=None, q_oversample=None, window=None):
     return tfr, dgr, gam
 
 
-if __name__ == '__main__':
+def main():
     from tftb.generators import fmconst
     import matplotlib.pyplot as plt
+
     sig = np.r_[fmconst(128, 0.2)[0], fmconst(128, 0.4)[0]]
-    ts = np.linspace(0, 1, 256)
-    tfr = ShortTimeFourierTransform(sig, timestamps=ts)
-    tfr.run()
-    tfr.plot(show_tf=True, cmap=plt.cm.viridis)
+
+    nr_samples = 256
+    n_fbins = nr_samples
```

Comment:
The `nr_samples` variable seems redundant.

Reply:
Oops. Yes I'll correct that.